custom_components/tuya_local/devices/jiahong_et72w_thermostat.yaml

```diff
@@ -219,3 +217,34 @@ entities:
             value: "7 (Adaptive)"
           - dps_val: 3
             value: "5+1+1 (Adaptive)"
+  - entity: select
+    category: config
+    name: Sensor Type
+    icon: "mdi:heating-coil"
+    dps:
+      - id: 116
+        type: integer
+        name: option
+        unit: W
```

Comment:
Unit is not used by the select entity. It would be better to add it as a suffix on the values. Rather than "Sensor type" this seems to be "Heating power".

Reply:
Renamed the sensor to align with the app naming and aligned the icon